core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -48,7 +48,8 @@ require('services/context.service.ts');
 require('services/editability.service.ts');
 require('services/id-generation.service.ts');
 require('services/user.service.ts');
-const WaveSurfer = require('third-party-imports/wave-surfer.import.ts');
+
+import WaveSurfer from 'wavesurfer.js';
```

Comment:
Do wee need parenthesis around `WaveSurfer`?
Also, can you please share the image of the audio waves in the translation tab after this change?

Reply:
No we don't need parenthesis.

Here is an image
![Screenshot_20200510_032047](https://user-images.githubusercontent.com/36989112/81485814-c3114180-926d-11ea-9e45-fcecaec81c31.png)